File: Selenium/Project/Activity1.java
package project;
import static org.testng.Assert.assertEquals;

import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.testng.Assert;
import org.testng.annotations.AfterClass;
import org.testng.annotations.BeforeClass;
import org.testng.annotations.Test;



public class Activity1 
{
	WebDriver driver;
	@BeforeClass
	public void setup()
	{
		driver= new FirefoxDriver();
		driver.get("http://alchemy.hguy.co/orangehrm");

	}
	@Test
	public void test() {
		String title= driver.getTitle();
		Assert.assertEquals(title, "OrangeHRM");
	}
	@AfterClass
	public void teardown()
	{
		driver.quit();

	}


}

File: Selenium/Project/Activity1.py
from selenium import webdriver

with webdriver.Firefox() as driver:
    driver.get("http://alchemy.hguy.co/orangehrm")
    title = driver.title
    print("Title of the page is : "+title)

    assert title == "OrangeHRM"
    print("Tile of the page is matching")
    driver.quit()